scripts/helmer.sh

```diff
@@ -74,6 +74,26 @@ mtls_configmap() {
     kubectl get cm -n $NAMESPACE mtlscm &>/dev/null || kubectl create cm -n $NAMESPACE mtlscm --from-file=client-ca.crt=$(dirname "$0")/../resources/mtls/rootCA.crt
 }
 
+create_kw_namespace_with_psa() {
+    local ns_name="$1"
+    local enforce_version=$(kubectl version | grep "Server Version" | awk '{print $3}' | awk -F'[.+]' '{print $1 "." $2}')
+
+    if [[ -z "$ns_name" ]]; then
+        echo "Usage: create_namespace_with_psa <ns_name>
+        return 1
+    fi
```

Comment:
We use `set -u` on top of the script (Treat  unset variables as an error).
You create variable with `ns_name="$1"` which will fail if $1 is unset. ($1: unbound variable)

This check `if [[ -z "$ns_name" ]]; then` is unreachable.

```suggestion
```